### ISSUE TITLE
Probably bug

### ISSUE THREAD

**Issue by saspol** [alextaujenis/RBD_Light#4]
Hello,
when i use construction like this:

``` arduino
void loop() {
   LED.update();
   if ( case1 ) {
      LED.setBrightnessPercent(50);
   } else if (case2) {
      LED.blink(40, 35);
   }
}
```

If case2 happens _after_ case1, blink does not work, LED still 50% brightness.
To get it work need to add additional LED.off(); inside loop. This makes everything complicated.
I think this is wrong behaviour.

**Comment by alextaujenis** [comment]
Yes, this is a bug. Setting the brightness first actually does prohibit the light from blinking unless you call light.off(). I'll patch and push a fix. Thank you for reporting this.

**Comment by alextaujenis** [comment]
Bug fix released in [v2.1.6](https://github.com/alextaujenis/RBD_Light/releases/tag/2.1.6)
